### ISSUE TITLE
Dropbox automatically create empty config and delete all my profiles

### ISSUE THREAD

**Issue by riskikukuh** [niceit/tabby-cloud-sync-settings#61]
**OS**: Ubuntu 20.04.2 LTS (Focal Fossa)
**Tabby**: v1.0.216
**Tabby Cloud Sync Settings**: v1.6.5

**Issue**
I installed the plugin because I wanted to make it easier when switching devices. I want all my profiles (ssh accounts) and vaults to sync when I switch to another device.

I use “Dropbox” as the Cloud Provider. When I enable Sync on the main device, all profiles and vaults on the main device are lost / deleted. And when I check on Dropbox, the sync file is only 8KB. 

I have tried several times, and the result remains the same. Is that how Cloud Sync works?

Thank you

**Comment by niceit** [comment]
Make sure you have not enabled the vault password before using this plugin.

**Comment by riskikukuh** [comment]
But i want to sync my vault also. Is it make the plugin crash ?

**Comment by niceit** [comment]
yes, as I answered, if you use the master password for config, consider not using this plugin because it's not supported yet.

**Comment by riskikukuh** [comment]
Okey. I watched the logs yesterday. And i found the Server Updated is always greater than Local Updated. I change something in local ( config.yaml ) but it still sync Cloud to Local

**Comment by niceit** [comment]
When you saved new changes, it will be lost?
